boms/standard-ee/pom.xml

```diff
@@ -2495,6 +2495,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.narayana.xts</groupId>
+                <artifactId>byteman_support</artifactId>
```

Comment:
I this used in production code? AFAICT it is not.

If not please remove this and restore the section of the root pom that touches it and @parsharma will handle this dependency as part of #16471 / https://issues.redhat.com/browse/WFLY-17412.

Reply:
Sorted! Thanks Brian!